DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Administrator/AdministratorController.cs

```diff
@@ -11,26 +14,43 @@
     [Route("TrackingSystem/Centre/Administrators")]
     public class AdministratorController : Controller
     {
+        private readonly ICommonService commonService;
         private readonly IUserDataService userDataService;
 
-        public AdministratorController(IUserDataService userDataService)
+        public AdministratorController(
+            IUserDataService userDataService,
+            ICommonService commonService
+        )
         {
             this.userDataService = userDataService;
+            this.commonService = commonService;
         }
 
         [Route("{page=1:int}")]
         public IActionResult Index(

```

Comment:
Does pressing the back button take the user to the previous page in pagination with JS off? I would assume it works similarly to the searching, where updating the search (now filter) resets the page to the first, so if you are ever beyond page 1, the previous page is definitely reached by pressing back, but worth double checking.

Reply:
It takes you to the previous query string version e.g. if you go back after applying a filter it will remove it. There is a new subtask that I hadn't created yet to look into whether we can change this behaviour. (I suspect not)